habitat-lab/habitat/core/vector_env.py

```diff
@@ -480,6 +480,9 @@ def close(self) -> None:
 
         self._is_closed = True
 
+        # Force the batch renderer to release its context.
+        self._batch_renderer = None
```

Comment:
The tests were leaking the graphics context because `VectorEnv` would not clear the batch renderer shared pointer.

Reply:
This looks odd. There is no guarantee of Python actually deleting this object when the references go to zero, right? How about an explicit `close()` on the batch renderer?